internal/kafka/internal/handlers/cloud_providers.go

```diff
@@ -75,9 +77,15 @@ func (h cloudProvidersHandler) ListCloudProviderRegions(w http.ResponseWriter, r
 					cloudRegion.Enabled = true
 					cloudRegion.SupportedInstanceTypes = region.SupportedInstanceTypes.AsSlice()
 					for _, instType := range cloudRegion.SupportedInstanceTypes {
+						// --- to be removed once MaxCapacityReached has been removed. ---
 						maxCapacityReached := true
 						kafka.InstanceType = instType
 						kafka.Region = cloudRegion.Id
+						size, e := h.kafkaConfig.GetFirstAvailableSize(instType)
+						if e != nil {
+							return nil, errors.NewWithCause(errors.ErrorGeneral, err, "Unable to list cloud provider regions")
```

Comment:
the error message doesnt seem correct when , err is used instead of e

Reply:
:+1: that's correct. I added a new commit with the fix